### PR TITLE
Task2.1

### PR DESCRIPTION
**Pull request by huhsu1**
added issues to NewsItems.

In NewsItems, now there is a drop down menu asking for issues.

Also created extra column in NewsItems to save such things.

### CODE REVIEW

File: app/models/news_item.rb
Comment (huhsu1) [inline app/models/news_item.rb:1]: added possible items for issues.

File: app/views/my_news_items/_form.html.haml
Comment (huhsu1) [inline app/views/my_news_items/_form.html.haml:1]: adds issues as possible form in a drop down menu in creating new news_items.

File: db/migrate/20231129080702_add_issue_to_news_item.rb
Comment (huhsu1) [inline db/migrate/20231129080702_add_issue_to_news_item.rb:1]: adding column to news_item to save issues.

**Review by KEV-INZHU — APPROVED** [review]
look good to me

**Pull request merged**